test-config/wdio.browserstack.conf.js

```diff
@@ -9,27 +9,25 @@ const commonCapabilities = {
   'browserstack.appium_version': '1.14.0'
 }
 
+// https://www.browserstack.com/automate/capabilities
 const allCapabilities = [
-  { ...commonCapabilities, browserName: 'Chrome', browser_version: '89.0', os: 'Windows', os_version: '10' },
-  { ...commonCapabilities, browserName: 'Chrome', browser_version: '74.0', os: 'Windows', os_version: '10' },
+  { ...commonCapabilities, browserName: 'Chrome', browser_version: 'latest', os: 'Windows', os_version: '10' },
   { ...commonCapabilities, browserName: 'Chrome', browser_version: '40.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'firefox', browser_version: '87.0', os: 'Windows', os_version: '10' },
-  { ...commonCapabilities, browserName: 'firefox', browser_version: '67.0', os: 'Windows', os_version: '10' },
+  { ...commonCapabilities, browserName: 'firefox', browser_version: 'latest', os: 'Windows', os_version: '10' },
   { ...commonCapabilities, browserName: 'firefox', browser_version: '52.0', os: 'Windows', os_version: '7' },
-  { ...commonCapabilities, browserName: 'Safari', browser_version: '14.0', os: 'OS X', os_version: 'Big Sur' },
+  { ...commonCapabilities, browserName: 'Safari', browser_version: '14.1', os: 'OS X', os_version: 'Big Sur' },
```

Comment:
the Mac and iPhone mostly have 1 browser per os version (`.0` and `.1` versions actually). I prefer to show them explicitly rather than using the `latest`. `latest` in this situation would always be one version anyway.